Source/WebCore/bindings/js/StructuredClone.cpp

```diff
@@ -77,29 +77,65 @@ JSC_DEFINE_HOST_FUNCTION(structuredCloneForStream, (JSGlobalObject* globalObject
 
     JSValue value = callFrame->uncheckedArgument(0);
 
-    if (value.inherits<JSArrayBuffer>())
-        RELEASE_AND_RETURN(scope, cloneArrayBufferImpl(globalObject, callFrame, CloneMode::Full));
```

Comment:
I take it that `cloneArrayBufferImpl` needs to stay as it is?

Reply:
Right now, we leave `cloneArrayBufferImpl` since it is only used by ReadableByteStream, which is incomplete work, so we don't know whether this change is necessary for that.